src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Security/TokenSecuredKnownUser/TokenSecuredKnownUser.xaml.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright 2016 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
```

Comment:
As mentioned above seems to be some indentation anomalies, particularly in regards to comments positioning in this file